org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/ParentProjectDescriptor.xtend

```diff
@@ -69,7 +70,7 @@ class ParentProjectDescriptor extends ProjectDescriptor {
 	}
 	
 	def String getTychoVersion() {
-		'1.2.0-SNAPSHOT'
+		'1.1.0'
```

Comment:
I would expect the condition here:
if (config.javaVersion == JavaVersion.JAVA10) '1.2.0' else '1.1.0'

Reply:
WDYT?

Reply:
Won’t catch the comment

Reply:
Since we throw this away anyway ....